gui-backend/src/main/java/com/rln/gui/backend/implementation/methods/AutoapproveApiImpl.java

```diff
@@ -80,6 +84,25 @@ public List<LedgerAddressDTO> get() {
     return result;
   }
 
+  public List<WalletAddressDTO> getForWallet() {
+    var accounts = accountCache.getAccounts();
+    var remoteOwnedAddresses = remoteOwnedAddressSupplier.getRemoteOwnedAddresses();
+    ArrayList<WalletAddressDTO> result = new ArrayList<>(remoteOwnedAddresses.size() + accounts.size());
+
+    result.addAll(remoteOwnedAddresses);
+    for (var account : accounts) {
+      var partySetlId = setlPartySupplier.getSetlId(account.getProvider());
+      result.add(WalletAddressDTO.builder()
+          .address(account.getIban())
+          .partyId(partySetlId)
+          .bearerToken("DummyToken") // no one actually checks this
```

Comment:
Can't this be used from the input?
If not, define it as `private static final String defaultBearerToken = "DummyToken"` on class level.

Reply:
Done